htdocs/user/card.php

```diff
@@ -1959,18 +1959,18 @@
 				}
 				print '</td></tr>';
 			}
-
-			print '<tr class="nooddeven"><td>'.$langs->trans("LastConnexion").'</td>';
-			print '<td>';
-			if ($object->datepreviouslogin) {
-				print dol_print_date($object->datepreviouslogin, "dayhour", "tzuserrel").' <span class="opacitymedium">('.$langs->trans("Previous").')</span>, ';
-			}
-			if ($object->datelastlogin) {
-				print dol_print_date($object->datelastlogin, "dayhour", "tzuserrel").' <span class="opacitymedium">('.$langs->trans("Currently").')</span>';
+			if ((dolibarr_get_const($db, "MAIN_ENABLE_LOGINS_PRIVACY", $conf->entity) == 0) || (dolibarr_get_const($db, "MAIN_ENABLE_LOGINS_PRIVACY", $conf->entity) == 1 && $object->id == $user->id)) {
```

Comment:
dolibarr_get_const is used by the master.inc.php to load config. Once done, you must use
getDolglobalconst('MAIN_ENABLE_LOGINS_PRIVACY')